state/relationunit.go

```diff
@@ -304,6 +304,7 @@ func (ru *RelationUnit) LeaveScope() error {
 	// Destroy changes the Life attribute in memory (units could join before
 	// the database is actually changed).
 	desc := fmt.Sprintf("unit %q in relation %q", ru.unitName, ru.relation)
+	logger.Debugf("%v leaving scope", desc)
```

Comment:
Just checking that this was deliberately left.

Reply:
Yeah it was, we need a message here to let us know a unit has left scope